generative/maisi/configs/label_dict_124_to_132.json

```diff
@@ -0,0 +1,502 @@
+{
```

Comment:
May I what this config file is for? Can we unify it to the `label_dict.json`?

Thanks.

Reply:
In my understanding, it will involve much work to do? @guopengf